src/fqdncache.h

```diff
@@ -11,19 +11,17 @@
 #ifndef SQUID_SRC_FQDNCACHE_H
 #define SQUID_SRC_FQDNCACHE_H
 
-#include "ip/Address.h"
+#include "dns/forward.h"
 #include "sbuf/forward.h"
 
 class StoreEntry;
```

Comment:
If we are going to remove one forward declaration duplication in a PR dedicated to this kind of maintenance, let's remove all duplicated forward declarations in this file (at least) that can be easily de-duplicated:

```suggestion
#include "sbuf/forward.h"
#include "store/forward.h"

```